packages/@aws-cdk/aws-scheduler-targets-alpha/lib/stepfunctions-start-execution.ts

```diff
@@ -0,0 +1,35 @@
+import { IScheduleTarget, ISchedule } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { IStateMachine } from 'aws-cdk-lib/aws-stepfunctions';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * Use an AWS Step function as a target for AWS EventBridge Scheduler.
+ */
+export class StepFunctionsStartExecution extends ScheduleTargetBase implements IScheduleTarget {
```

Comment:
I would recommend to add unit tests for this target too. See https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-scheduler-targets-alpha/test/target.test.ts#L9. We probably need to rename that test file and create one file with unit tests per each target.

Reply:
I created a new unit test which contains almost identical unit tests for step functions.

I also renamed the existing file `target.test.ts` into `lambda-invoke.test.ts`.